api/lib/infrastructure/serializers/jsonapi/membership-serializer.js

```diff
@@ -87,6 +87,7 @@ module.exports = {
     return new Membership({
       id: json.data.id,
       organizationRole: json.data.attributes['organization-role'],
+      disabledAt: json.data.attributes['disabled-at'],
```

Comment:
C'est peut être pas le bon endroit pour lever la remarque, mais la date de désactication est envoyée par le front. Est-ce un choix par défaut ? Je me dis que si le front envoie un attribut booléen (ex: active = false), et que le back alimente le champ updatedAt avec la date locale, il y a moins de risque d'incohérence (ex: timezone) ou de sécurité (ex: curl avec une date dans le passé pour faire croire à une action qui n'a pas eu lieu)

Reply:
L'archivage des campagnes fonctionne comme ça: le endpoint exposé ne réalise que cette action, et il n'a pas de paramètre (à part campaignId)
api/lib/domain/usecases/archive-campaign.js
` return campaignRepository.update({ id: campaignId, archivedAt: new Date() });`

Reply:
Oui, j'allais plussoyer ta remarque sur le front qui envoie une date de désactivation, en ajoutant que ce serait encore plus correct d'avoir une action explicite pour ça… et tu as trouvé l'exemple parfait, well done 👍

Reply:
tribute to @sroccaserra : "codebase is the implicit standard"

Reply:
Un nouvel endpoint a été créé pour gérer la désactivation, comme disucté.